src/style.css

```diff
@@ -1 +1,73 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir-Book.ttf) format('opentype') tech(color-COLRv1);
+}
+
+body {
+  margin: 0;
+  padding: 20px;
```

Comment:
Consider using only top or bottom padding to avoid potential margin collapse issues. The checklist suggests being consistent with margins and padding.